frontend/src/views/pages/inventory/InventoryAdd.vue

```diff
@@ -224,6 +226,9 @@ export default {
         })
         .catch((error) => {
           console.log(error);
+          this.showMessage("Please select a Dealership", "danger")
+          //alert("stop");
+          //this.showMessage("Please select a Dealership", "danger")
           this.$router.replace("/pages/404");
```

Comment:
Remove the commented code and the 404 redirection. We don't need to redirect because we are showing the error message

Reply:
I removed the comments and the 404 redirection.